src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -870,6 +864,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     </PropertyGroup>
   </Target>
 
+  <Target Name="_RazorSdkCustomCollectWatchItems">
+    <ItemGroup>
+      <Watch Include="%(Content.FullPath)" Condition="'%(Content.Extension)' == '.razor' AND '%(Content.Watch)' != 'false'" />
```

Comment:
Is there a reason that this isn't `Content.Watch == 'true'`?

As in, I would expect `Content.Watch` to be an empty value if it is not set.

Reply:
> As in, I would expect Content.Watch to be an empty value if it is not set.

Yup. By default, this value is never set. You can opt individual files out of watch by setting them to `false`

Reply:
```suggestion
      <Watch Include="%(Content.FullPath)" Condition="'%(Content.Extension)' == '.razor' AND '%(Content.Watch)' != 'false'" />
      <Watch Include="%(Content.FullPath)" Condition="'%(Content.Extension)' == '.razor.css' AND '%(Content.Watch)' != 'false'" />
```